### DISCUSSION TITLE
I see!

### DISCUSSION THREAD

**Discussion by DiamondElement** [thatskymod/Sky-CotL-Scripts#110] [General]
I just saw that you included TSM!!

**Comment by thatskymod** [comment]
Yes, after allowing some time for the project to mature and initial issues to be resolved, I was able to test and share a well-functioning version of ThatSkyMod[^1] (not @thatskymod, heh).

Thank you for your continued support and encouragement.

[^1]: View developer's [GitHub page](https://github.com/TheSR007).
